database/tables/questions.pg

```diff
@@ -12,6 +12,7 @@ columns
     external_grading_image: text
     external_grading_timeout: integer
     grading_method: enum_grading_method not null default 'Internal'::enum_grading_method
+    hide_correct_answer: boolean default false
```

Comment:
For consideration: we use `show...` elsewhere, e.g. `showClosedAssessment` and `showClosedAssessment` in `infoAssessment.json`. There might be value in being consistent and doing the same here?

Reply:
I think it's used in enough places that it's worth being consistent. Will switch over.